Tests/SentryTests/Integrations/Session/SentrySessionTrackerTests.swift

```diff
@@ -80,6 +80,8 @@ class SentrySessionTrackerTests: XCTestCase {
         
         assertInitSessionSent()
         assertSessionStored()
+
+        XCTFail("Test failure")
```

Comment:
Is this still needed for testing or can it be removed now?

Reply:
Now it can, I had to check to make sure the script failed correctly, which it does now as of https://github.com/getsentry/sentry-cocoa/pull/2089/commits/c55d841e507bbbc2262478738770831f5f16d043